### DISCUSSION TITLE
Error message in disassembly view

### DISCUSSION THREAD

**Discussion by coronax** [prb28/vscode-amiga-assembly#273] [Q&A]
Hi everyone! I've just been using the extension for a couple of days, and really like it, but I am having a problem with the disassembly view and disassembled memory view in the debugger. The picture shows the error message I'm seeing in those views (unfortunately it won't let me copy the text itself). I'm wondering if there's anything I'm missing or need to configure to get this working. Does anyone have any ideas? Thanks in advance for the help!

![image](https://github.com/prb28/vscode-amiga-assembly/assets/12867087/3c2b116a-d757-4dba-bf91-fda9350b833f)

**Comment by Real-MJoe** [comment]
Unfortunately it's the same here. I thought it was one of my own cofingurations. I use VS Code 1.82 with Amiga Assembly
v1.8.3.

![grafik](https://github.com/prb28/vscode-amiga-assembly/assets/40276633/781b78c0-3f3f-456e-a410-6ba8af8025d9)

The “Memory Breakpoint” feature also doesn’t work. :/  I use Win-UAE for debugging and setting memory breakpoints.

**Comment by Real-MJoe** [comment]
Okay found some error logs:
![grafik](https://github.com/prb28/vscode-amiga-assembly/assets/40276633/33a5e1c8-4886-49c7-9fc8-7dea42ce1f8c)

![grafik](https://github.com/prb28/vscode-amiga-assembly/assets/40276633/b8b95348-dd19-43c7-9ae9-5c9e5b9406da)

**Reply by prb28** [reply]
It looks like a bug, I'll check it.

**Reply by Real-MJoe** [reply]
Is there any news yet? Can I help you?

**Comment by Real-MJoe** [comment]
Hmmmm.....this call don`t work, but why.
![grafik](https://github.com/prb28/vscode-amiga-assembly/assets/40276633/9f7cbe6e-2492-4944-b66c-8bf4f1815462)

here the right error:
![grafik](https://github.com/prb28/vscode-amiga-assembly/assets/40276633/4c303a1a-6214-4445-9d50-c462e93ccb26)

**Reply by prb28** [reply]
I think there is a bug in uae-dap. I've tried to debug, but debugging the two projects is not easy. @grahambates what is your way to debug it ?

**Reply by grahambates** [reply]
Yeah this is a problem with an incompatibility with node 18 and the wasm cstool build. I've had the same big on another project and have a fix. I'll sort it.

**Reply by pauljsymonds** [reply]
I'm happy to test anything required on the Mac M1 if that helps.

I may need some pointers to get the env set up if I need to compile and build.

**Reply by grahambates** [reply]
I've opened a PR to update to the fixed uae-dap version: https://github.com/prb28/vscode-amiga-assembly/pull/275

@pauljsymonds you're welcome to test. The easiest way is probably to check out that branch, open it in VSCode and 'Run and Debug: extension'. There's nothing Mac M1 specific though, as the change is to the WASM cstool build.

**Reply by Real-MJoe** [reply]
Nice one! I can confirm that the branch is working. :)
![grafik](https://github.com/prb28/vscode-amiga-assembly/assets/40276633/93cc2254-24d9-4dd5-8839-76b36db980f2)
